install_config/aggregate_logging.adoc

```diff
@@ -3628,3 +3636,304 @@ cluster. Edit your non-cluster logging service (for example, `logging-es`,
 ----
 $ oc patch svc/logging-es -p '{"spec":{"selector":{"component":"es","provider":"openshift"}}}'
 ----
+
+[[optimal-state-cluster]]
+=== Cluster not in an optimal state
+
+The reasons that your cluster may not be considered to be in an optimal state are
+that the cluster health is not `green` and/or the cluster size specified in the
+cluster health does not match the number of pods running for that cluster (e.g.
+not all nodes have joined the cluster).
+
+To check which currently applies to your cluster:
+
+. Get the name of all pods for your particular cluster
++
+----
+$ oc get pods -l component=es[-ops] -o name
+----
+
+. Check the health of the cluster using one of the pods found above
+----
+$ oc exec -c elasticsearch <es_pod> -- es_cluster_health
+----
+
+The total number of nodes in the cluster displayed from the second command should
+match the number of pods returned in the first command.
+
+==== If the cluster is not `green`
```

Comment:
[[if-the-cluster-is-not-green]]
s/If the cluster is not `green`/If the Cluster is Not `Green`

Reply:
should it be `Green`? the elasticsearch output prints it out as all lowercase

Reply:
@ewolinetz How about _If the cluster is not in a green state_ ?
A few other headings include _state_.